pkg/controller/build/build_controller.go

```diff
@@ -23,6 +23,9 @@ import (
 
 var log = logf.Log.WithName("controller_build")
 
+// succeedStatus default status for the Build CRD
+const succeedStatus string = "Succeed"
```

Comment:
Could we call it `Succeeded` please - unless there's a prior precedence that is being used ?

Reply:
none precedence, I just amend on the same commit.